LongWarOfTheChosen/Localization/BetterSecondWaveSupport.int

```diff
@@ -1,3 +1,11 @@
+; LWOTC Needs Translation
 [EnableChosen SecondWaveOptionObject]
-Tooltip="Enables the Chosen, who will activate after you complete the first network tower mission"
+Tooltip="Enables the Chosen, who will activate after you complete the first network tower mission."
 Description="The Chosen: Enables the Chosen"
+; End Translation
+
+; LWOTC Needs Translation
+[AllowSameRankAbilities SecondWaveOptionObject]
+Tooltip="Allows XCOM soldiers to purchase multiple perks of the same rank with AP"
```

Comment:
OK, so why did my tooptip require a full stop, but your doesn't 😛

Reply:
Oops, all tooltips and descriptions should have punctuation to match the vanilla ones.